src/DotNet.Status.Web/Controllers/AzurePipelinesController.cs

```diff
@@ -233,31 +233,77 @@ private async Task ProcessBuildNotificationsAsync(Build build)
 
 {changesMessage}
 ";
+                    string issueTitlePrefix = $"Build failed: {build.Definition.Name}/{prettyBranch} {prettyTags}";
+                    bool updateExisting = repo.UpdateExisting;
 
-                    var newIssue =
-                        new NewIssue($"Build failed: {build.Definition.Name}/{prettyBranch} #{build.BuildNumber}")
-                        {
-                            Body = body,
+                    if (updateExisting)
+                    {
+                        RepositoryIssueRequest issueRequest = new RepositoryIssueRequest {
+                            Creator = "dotnet-eng-status[bot]",
```

Comment:
Hrm... I wonder if there is some way we can fetch this given the token we have, rather than having to hard code it.  To see if it's "us".  Especially since I think the staging one has a different name, which means this won't work in staging.

Reply:
We should be able to call https://docs.github.com/en/rest/reference/apps#get-the-authenticated-app that, which should give us our own name.

Reply:
I didn't know about that. Cool.

Reply:
Ok, did some shenanigans using the api. Let me know if that works, or we could move this to the settings.json and just hardcode the bot's name there/put it in key vault and get it from there.